lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -38,23 +38,6 @@ class CourseCardModel extends Backbone.Model {
     return desiredCourseRun;
   }
 
-  getCourseRunWithHighestGrade(grades) {
-    const allEnrolledCourseRuns = this.context.course_runs.filter(run => run.is_enrolled);
-    if (allEnrolledCourseRuns.length <= 1) {
-      return null;
-    }
-
-    allEnrolledCourseRuns.sort((a, b) => (grades[a.key] || 0) - (grades[b.key] || 0));
-    return allEnrolledCourseRuns[allEnrolledCourseRuns.length - 1];
-  }
-
-  updateCourseRunWithHighestGrade(grades) {
-    const courseRunWithHighestGrade = this.getCourseRunWithHighestGrade(grades);
-    if (courseRunWithHighestGrade) {
-      this.setActiveCourseRun(courseRunWithHighestGrade, this.context.user_preferences);
```

Comment:
Do you have a sense of which course run will be set as active under this new world order? Either from manual testing or from reading other logic?

Reply:
This is called specifically for completed courses to override the "defualt" selected course run in `getCourseRun` (course_card_view.js:30), so whatever course run was selected for the course when it was in progress will be the active course